enigma-js/test/Enigma.spec.js

```diff
@@ -101,37 +101,6 @@ describe('Enigma tests', () => {
         'e762e49ec912be61358d5e90bff56a53a0ed42abfe27e3');
     });
 
```

Comment:
@apalepu23 Why are these unit tests removed?

Reply:
resolved in e113574b951710d7490ce5b697d81dc85ff685b0